File: csharp/difference-of-squares/DifferenceOfSquares.cs
using System;
using System.Linq;

public static class DifferenceOfSquares
{
    public static int CalculateSquareOfSum(int max) => (int)Math.Pow((Enumerable.Range(1, max).Sum()),2);
    
    public static int CalculateSumOfSquares(int max) => Enumerable.Range(1, max).Aggregate(0, (total, i) => total + (int)Math.Pow(i, 2));

    public static int CalculateDifferenceOfSquares(int max)=> CalculateSquareOfSum(max) - CalculateSumOfSquares(max);
}

File: python/difference-of-squares/difference_of_squares.py
def square_of_sum(number: int) -> int:
    sum = 0

    for i in range(number):
        sum += i + 1
    
    return sum ** 2


def sum_of_squares(number: int) -> int:
    sum = 0

    for i in range(number):
        sum += (i+1) ** 2

    return sum


def difference_of_squares(number: int) -> int:
    return square_of_sum(number) - sum_of_squares(number)
